packages/editor/src/components/post-preview-button/index.js

```diff
@@ -207,12 +207,6 @@ export class PostPreviewButton extends Component {
 				{ this.props.textContent
 					? this.props.textContent
 					: _x( 'Preview', 'imperative verb' ) }
-				<VisuallyHidden as="span">
```

Comment:
It looks like there are a couple of other uses of `PostPreviewButton`, like in the `PostLockedModal`, and I think that only says 'Preview'.

An option could be to change the text everywhere to 'Preview in new tab', or make the VisuallyHidden text something that can be enabled/disabled through a prop..

Reply:
Great catch, Probably risky to change that phrasing because it will likely not scale to mobile interfaces with the length of the phrase. 

So we should probably restore the hint for that use case.

Reply:
Yep, this was, in a way, mentioned on the issue:
> Looking at the implementation, not sure what is the preferred approach. PostPreviewButton accepts a textContent prop and fallbacks to a link text Preview if none is passed. So the default text does need the visually hidden text, while a more meaningful text passed via textContent may not need it. Since this depends on the actual text passed via the prop, I'd tend to think the behavior can't be automated and maybe an additional, explicit, prop to render the visually hidden text would be the best option.